doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -777,6 +777,12 @@ In a Coq term, writing :g:`$x` is semantically equivalent to
 :g:`ltac2:(Control.refine (fun () => x))`, up to re-typechecking. It allows to
 insert in a concise way an Ltac2 variable of type :n:`constr` into a Coq term.
 
+Similarly variables of type `preterm` have an antiquotation
+
+.. prodn:: term += $$@lident
```

Comment:
This production is at LEVEL 200 of `term`?  Same question for `$@lident` a few lines above.  Please update if it's not at that level.